source/options/VisualsUISubState.hx

```diff
@@ -50,8 +50,8 @@ class VisualsUISubState extends BaseOptionsMenu
 			true);
 		addOption(option);
 
-		var option:Option = new Option('Health Bar Transparency',
-			'How much transparent should the health bar and icons be.',
+		var option:Option = new Option('Health Bar Visibility',
+			'How much visible should the health bar and icons be.',
```

Comment:
Should probably be changed to "How visible the health bar and icons are." or something similar.

Reply:
> Should probably be changed to "How visible the health bar and icons are." or something similar.

Thats what theres written
Plus I wanted to edit the minimum, so I just changed the wrong term